### ISSUE TITLE
Move Updating Reviews (PUT Endpoint) to `/reviews`

### ISSUE THREAD

**Issue by DiljotSG** [DiljotSG/Throne-Backend#177]
It doesn't make much sense to have the updating reviews endpoint at `/washrooms/<id>/reviews/<id>` when creating a review we should POST to `/washrooms/<id>/reviews` which makes sense. But when updating it should just be at `/reviews/<id>`. Currently, we are ignoring the washroom ID in the URL and just using the review ID. So this endpoint should be moved. Because of this, we should also move the DELETE endpoint (which we aren't implementing, but is still there as a placeholder) to `/reviews/<id>` as well. The POST API docs should be updated to reflect these changes.

To summarize:
- Create a new review: `POST /washrooms/<id>/reviews` (done ✅ by Aaron).
- Update an existing review `PUT /reviews/<id>` (needs to be moved).
- Delete an existing review `DELETE /reviews/<id>` (just a placeholder, but still needs to be moved to match the new location for the PUT endpoint above).
